source/release-notes/1.2.txt

```diff
@@ -0,0 +1,39 @@
+.. _c2c-release-notes-1.1:
+
+===================================
+Release Notes for ``mongosync`` 1.2
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.2 Released February 16, 2023
+
+Support for Older Versions of MongoDB
+-------------------------------------
+
+{+c2c-product-name+} 1.2 adds support for sync between clusters running 
+older (pre-6.0) releases of MongoDB Server.
+
+If you would like to use ``mongosync`` with an older version of MongoDB,
+contact Professional Services.
+
+
+Guardrails for Unsupported Versions of MongoDB
+----------------------------------------------
+
+``mongosync`` now returns an error if you call :ref:`c2c-api-start`
+to sync clusters of an unsupported release of MongoDB Server.
```

Comment:
I think it would be clearer to say "running an unsupported" rather than "of an unsupported".

Reply:
Fixed